code/datums/mind.dm

```diff
@@ -326,11 +326,11 @@
 							qdel(I)
 							break
 				H << "<span class='notice'><font size =3><B>Your loyalty implant has been deactivated.</B></font></span>"
-				log_admin("[key_name_admin(usr)] has de-loyalty implanted [current].")
+				log_admin(text="[key_name_admin(usr)] has de-loyalty implanted [current].",admin_key=key_name_admin(usr),ckey=key)

```

Comment:
For ref, this can be `key_name(current)`. Although, now that I think about it, (src.)key is fine too.